packages/flutter_tools/lib/src/tracing.dart

```diff
@@ -45,7 +46,8 @@ class Tracing {
       );
       try {
         final Completer<void> whenFirstFrameRendered = Completer<void>();
-        (await vmService.onExtensionEvent).listen((ServiceEvent event) {
+        await vmService.streamListen('Extension');
```

Comment:
Do we need to manually stop listening?

Reply:
I'm not sure (fyi @bkonyi ). I think this is only the client listening, so whenever we disconnect this goes away. We never unsubscribed before so I don't think we need to start in order to be compatible .

Reply:
You should be fine if you don't invoke `streamCancel` before taking down a VM service client. You only really need to cancel if you want to keep the client alive but no longer wish to receive events from a particular stream.